galley/mutations.py

```diff
@@ -69,3 +83,37 @@ def upsert_menu_data(args):
     mutation = build_upsert_mutation_query(args)
     response = make_request_to_galley(op=mutation)
     return validate_response_data(response)
+
+
+def build_update_mutation_query(args):
+    ids = []
+    if args["ids"] is not None and len(args["ids"]) > 0:
```

Comment:
```suggestion
    if args.get("ids"):
```
This will check for both cases. Also, if "ids" isn't actually in args, `args["ids"]` would still throw an error here so its best to use `.get()` for checking.